client/src/Game.cpp

```diff
@@ -84,31 +68,36 @@ void Game::processEvents() {
 }
 
 void Game::update(sf::Time deltaTime) {
-    player.update(deltaTime);
-
-    playerCommands.updatePlayer(deltaTime);
+    if (!player.expired()) {
```

Comment:
Before loading the player isn't set. Do you think I should remove this check on the weak_ptr and add an attribute `bool loaded` to the game class ?